sumologic/resource_sumologic_field.go

```diff
@@ -0,0 +1,131 @@
+package sumologic
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceSumologicField() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSumologicFieldCreate,
+		Read:   resourceSumologicFieldRead,
+		Update: resourceSumologicFieldUpdate,
```

Comment:
Since update is `nil` we can simply remote it instead. Similar to how it was in content: https://github.com/SumoLogic/terraform-provider-sumologic/blob/5501c3e68ff12e2bdf6a03a7d227e7247d09fbe8/sumologic/resource_sumologic_content.go